### DISCUSSION TITLE
Is it possible to export/import gTile configuration?

### DISCUSSION THREAD

**Discussion by wirehiker** [gTile/gTile#394] [Q&A] [answered]
Is it possible to export the configuration of Accelerators, Resize setup, Margins from one installation to another?

**Comment by scherepanov** [comment]
Not possible now.
Most likely will be one of next things to implement.
@Strauman do you have time to resume work on it?

**Comment by pbsds** [comment, marked as answer]
`dconf dump /org/gnome/shell/extensions/gtile/`

There is also a `load` command

**Comment by c-neto** [comment]
Thanks for your [comment](https://github.com/gTile/gTile/issues/121#issuecomment-657667892) @pbsds! Your comment really helped me :)

I would like to share with us, how I configured my gTile parameter values.

- I configured the gTile parameter in the Gnome menu `Extensions`;
- I dumped the configurations parameters:
```bash
$ dconf dump /org/gnome/shell/extensions/gtile/ > my-custom-gtile-configs.conf
```
- The output of the last command is the parameters value of the gTile extension:
```conf
$ cat my-custom-gtile-configs.conf

[/]
preset-resize-1=['<Super>KP_1']
preset-resize-2=['<Super>Down']
preset-resize-3=['<Super>KP_3']
preset-resize-4=['<Super>Left']
preset-resize-5=['<Super>Space']
preset-resize-6=['<Super>Right']
preset-resize-7=['<Super>KP_7']
preset-resize-8=['<Super>Up']
preset-resize-9=['<Super>KP_9']
theme='Default'
```
> Be careful with accelerators' key map conflicts. In my case, I needed to change some key maps of the `org/gnome/desktop/wm/keybindings` configuration (that is the same configuration of the Gnome menu `Settings > Keyboards > Shortcuts`)

- To load the parameters when necessary (in other computers for example), It is only necessary to execute:

```bash
$ dconf load /org/gnome/shell/extensions/gtile/ < my-custom-gtile-configs.conf
```

>> All keys available to be configured are in this code: https://github.com/gTile/gTile/blob/master/app.ts#L198-L287
